developer_guides/debugability/coredump-reader/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _dbg-coredump-reader:
+
+Coredump-reader
+###############
+
+Tool for processing FW stack dumps. In verbose mode it prints DSP registers,
```

Comment:
Lines 6-8: This might work better: "In verbose mode it prints the stack leading to the core dump including DSP registers and function calls. It outputs unwrapped gdb command function call addresses to human readable function call format either to a file or stdin (is this supposed to be stdout?) "

Reply:
Yes, it should be stdout, thanks